pilot/pkg/xds/sds.go

```diff
@@ -439,30 +439,21 @@ func toEnvoyTLSSecret(name string, certInfo *credscontroller.CertInfo, proxy *mo
 	}
 }
 
-func containsAny(mp sets.Set[model.ConfigKey], keys []model.ConfigKey) bool {
```

Comment:
nit: I don't think everything needs to be a method, if its a niche one-off use case a local function like this is fine.